### PR TITLE
task to remove workflow from a project

### PR DESCRIPTION
**Pull request by boutell**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -694,6 +695,30 @@ For pages, it is almost as straightforward. Click on "Pages" in the admin bar to
 
 As with pieces, change "Trash" to "No." The "Trash" field will be located right after the "Published" field. When you click save, the page will be live in this locale.
 
+## Removing workflow from a project
```
Comment (abea) [inline README.md]: We should include some additional steps. The main thing I see is the warning to remove from app.js before starting the site back up. We should have a warning that down-time is probably needed. So what are the steps on a production site?

1. Plan for down time.
2. Run the task on the server
3. Update your project to disable workflow (and probably any related code) and deploy.
4. Start the site back up.

File: lib/tasks.js [hunk 2]
```diff
@@ -425,6 +426,70 @@ module.exports = function(self, options) {
...
+          workflowModified: 1,
+          workflowLocaleForPathIndex: 1,
+          workflowMoved: 1,
+          workflowMovedIsNew: 1
+        }
+      }, {
+        multi: true
+      });
+    }).then(function() {
+      console.log('Removing the commits collection...');
```
[the remark this answers is not in the document]
Comment (boutell) [reply lib/tasks.js]: Not in a task. Tasks are actually intended to communicate directly with the terminal.

File: lib/tasks.js [hunk 3]
```diff
@@ -425,6 +426,70 @@ module.exports = function(self, options) {
...
+    }
+    var keep = locale + (argv.draft ? '-draft' : '');
+    return Promise.try(function() {
+      console.log('Making sure the content you have chosen to keep exists...');
+      return self.apos.docs.db.count({
+        workflowLocale: keep
+      });
+    }).then(function(count) {
+      if (!count) {
+        throw 'There is no content with workflowLocale set to ' + keep + '.\nI think you chose the wrong locale or workflow was never set up. Exiting.';
```
[the remark this answers is not in the document]
Comment (boutell) [reply lib/tasks.js]: It says "exiting."

[the remark this answers is not in the document]
Comment (boutell) [reply lib/tasks.js]: Fixing the other thing.

**Comment by boutell** [comment]
Thanks, I implemented your notes.

**Pull request merged**